ui/installer/VCSA/configs

```diff
@@ -10,3 +10,6 @@ VIC_UI_HOST_URL="NOURL"
 
 # If VIC_UI_HOST_URL starts with "https", enter SHA-1 thumbprint of the web server hosting the vic-ui.zip file
 VIC_UI_HOST_THUMBPRINT=""
```

Comment:
@stuclem it would be great if you could also make a note in the documentation that the fingerprint should be delimited by colons not space. google chrome gives a fingerprint that has spaces in it, which doesn't work with VC. so i've made a note in the `configs` file both for VCSA and vCenter for Windows.